pallets/runtime/src/mips.rs

```diff
@@ -76,6 +76,8 @@ pub struct MipsMetadata<BlockNumber: Parameter, Hash: Parameter> {
     end: BlockNumber,
     /// The proposal being voted on.
     proposal_hash: Hash,
+    /// The proposal url for proposal discussion.
+    url: Vec<u8>,
```

Comment:
If I remember correctly we are providing the dedicated type for Vec<u8> I think @vkomenda did that

Reply:
What are dedicated types ? Not seeing much on them online can you point me in the right direction ?

Reply:
Dedicated type is not a terminology, I am just referencing it. You can check out the `TokenName, AssetId` type present in the `asset.rs`